flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnSessionCli.java

```diff
@@ -596,11 +596,11 @@ public int run(String[] args) throws CliArgsException, FlinkException {
 					//------------------ ClusterClient deployed, handle connection details
 					yarnApplicationId = clusterClient.getClusterId();
 
-					String jobManagerAddress =
-						clusterClient.getJobManagerAddress().getAddress().getHostName() +
-							':' + clusterClient.getJobManagerAddress().getPort();
+//					String jobManagerAddress =
+//						clusterClient.getJobManagerAddress().getAddress().getHostName() +
```

Comment:
commented temporarily because it does not work in flip6

Reply:
had to add it back because `YARNSessionCapacitySchedulerITCase` is relying on it